pdns/dnsdistdist/doh.cc

```diff
@@ -655,15 +655,15 @@ static void doh_dispatch_query(DOHServerConfig* dsc, h2o_handler_t* self, h2o_re
   }
 }
 
-static bool getHTTPHeaderValue(const h2o_req_t* req, const std::string& headerName, string_view& value)
+static bool getHTTPHeaderValue(const h2o_req_t* req, const std::string& headerName, boost::string_view& value)
 {
   bool found = false;
   /* early versions of boost::string_ref didn't have the ability to compare to string */
-  string_view headerNameView(headerName);
+  boost::string_view headerNameView(headerName);
```

Comment:
Well, we really would like to use `std::string_view` when available.. How does lmdb-safe.hh compiles on OpenBSD? Because the `string_view` detection is a copy-paste from that code..

Reply:
I normally don't build lmdb support on OpenBSD. But indeed, it gives similar errors.